### DISCUSSION TITLE
결제 페이지 - 선물내역 컴포넌트 설계

### DISCUSSION THREAD

**Discussion by uraflower** [KakaoFunding/front-end#108] [General]
![image](https://github.com/KakaoFunding/front-end/assets/82873315/b17e630d-c337-4f11-ad70-16155615644d)

> 빨간 박스 부분을 "선물내역", 파란 박스 부분을 "상세정보"라고 하겠습니다.

## 타이틀
이 페이지는 결제 페이지입니다. 선물 뿐만 아니라 펀딩 결제도 이 페이지에서 이루어집니다.
펀딩 결제할 때에도 "선물내역"이라는 타이틀을 사용하는 게 적절한지 의문이 들어서 다른 분들의 의견을 듣고 싶습니다.

## 상세정보
또 상세정보 부분 내용도 상품을 "선물"한다는 점에 초점이 맞춰져 있어요.
예를 들면 "상품금액", "선택수량", "수신인원"은 펀딩 결제에서는 필요하지 않은 정보예요.
그래서 다른 내용을 넣는 게 좋다고 생각이 듭니다.
아니면 상세정보 부분을 과감하게 삭제하고, 결제금액만 보여주는 방법도 있습니다. (기존에는 결제금액을 클릭해야 상세정보 영역이 펼쳐지거나 접히는 구조)

---
다른 분들께서는 **컴포넌트의 타이틀**과 **상세정보 내용**에 대해 어떻게 생각하시는지 의견을 듣고 싶습니다!
(+ 컴포넌트 구현 이슈(#109)에는 일단 기존 페이지 기준으로 작성해놓겠습니다)

**Comment by devkyoung2** [comment]
1. 펀딩내역이 나을 것 같습니다.
2. 상세정보 삭제가 나을 것 같습니다.
